src/pages/Account/hooks/useLogEventWithBasic.ts

```diff
@@ -0,0 +1,23 @@
+import {useWallet} from "@aptos-labs/wallet-adapter-react";
+import {Statsig} from "statsig-react";
+import {useGlobalState} from "../../../global-config/GlobalConfig";
+
+// Log event with basic info
+export const useLogEventWithBasic = () => {
```

Comment:
lgtm! would you mind updating the codebase to also use this hook for event loggings as well?

Reply:
OK, let's do that in another PR.